source/connect/authentication-connection.txt

```diff
@@ -13,3 +13,162 @@ Authentication Connection Tab
    :class: singlecol
 
 
+The :guilabel:`Authentication` tab allows you to connect to deployments that 
+require authentication. To learn about authentication mechanisms within MongoDB,
+see :manual:`Authentication Mechanisms </core/authentication/#authentication-mechanisms>`.
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`Authentication` tab.
+
+      Select your authentication method from the following options: 
+      
+      - :ref:`Username / Password <username-password>`
+      
+      - :ref:`X.509 <x509>`
+      
+      - :ref:`Kerberos <kerberos-authentication>`
+      
+      - :ref:`LDAP <ldap-authentication>`
+      
+      - :ref:`AWS IAM <aws-iam-authentication>`
+
+      .. _username-password:
+
+      Username / Password 
+      ~~~~~~~~~~~~~~~~~~~
+
+      Provide the following information:
+
+      - Username
+
+      - Password
+
+      - (Optional) :manual:`Authentication Database </core/security-users/#authentication-database>`
+
+      - Authentication Mechancism:
+         
+        - Default
+
+          The ``Default`` setting selects the first authentication mechanism 
+          supported by the server according to an order of preference. For 
+          more information on the order of preference, see :driver:`Default 
+          </node/current/fundamentals/authentication/mechanisms/#default>`
+
+        - :manual:`SCRAM-SHA-1 </core/security-scram/>`
+
+        - :manual:`SCRAM-SHA-256 </core/security-scram/>`
+
+      .. _x509:
+
+      X.509
+      ~~~~~
+
+      Select :guilabel:`X.509` if the deployment uses :manual:`X.509 
+      </core/security-x.509/>` as its authentication mechanism.
+
+      .. note:: 
+
+         X.509 Authentication requires a client certificate. To enable 
+         TLS and add a client certificate, see the :ref:`TLS / SSL tab <tls-ssl-tab>`
+         in |compass-short|.
+
+      .. _kerberos-authentication:
+
+      Kerberos
+      ~~~~~~~~
+
+      Select :guilabel:`Kerberos` if the deployment uses :manual:`Kerberos 
+      </core/kerberos/>` as its authentication mechanism. 
+
+      Provide the following information:
```

Comment:
[question]

Not trying to expand scope too much, but what do you think about possibly organizing these items into a table with a column for the field and a column for a description? I'm noticing that we have a couple fields where we're providing more detailed explanations, which is super helpful, but it makes me wonder if it would be easier to parse in a table format.

Reply:
I agree and updated the list to be a table.